contrib/terraform/gcp/variables.tf

```diff
@@ -0,0 +1,257 @@
+
```

Comment:
In a number of variables there is: `default = ""`, that makes the variable optional, but I think that they are often required or that an appropriate default is available. Can you double check the defaults provided for variables?

Reply:
@vladwa can you double check this?